examples/benchmarks/pylot/pylot_log_analyzer.py

```diff
@@ -0,0 +1,107 @@
+import csv
+from absl import app
+from absl import flags
+
+import erdos.graph
+from erdos.operators import CountWindowTrigger, FileWriterOp, WindowOp, Window, WindowAssigner, WindowProcessor
+from erdos.data_stream import DataStream
+from erdos.message import Message
+from erdos.op import Op
+from erdos.timestamp import Timestamp
+
+FLAGS = flags.FLAGS
+flags.DEFINE_string('framework', 'ros',
+                    'Execution framework to use: ros | ray.')
+flags.DEFINE_string('front_camera_locations',
+                    'front_left,front_center,front_right',
+                    'Comma-separated list of locations')
+
+
+class LogReaderOp(Op):
+    def __init__(self, name, output_name, log_file_name):
+        super(LogReaderOp, self).__init__(name)
+        self._output_name = output_name
+        self._log_file_name = log_file_name
+
+    @staticmethod
+    def setup_streams(input_streams, output_name):
+        return [DataStream(name=output_name)]
+
+    def execute(self):
+        output_stream = self.get_output_stream(self._output_name)
+        with open(self._log_file_name, 'r') as log_file:
+            csv_reader = csv.reader(log_file)
+            for row in csv_reader:
+                name = row[0]
+                processing_time = float(row[1])
+                coordinates = [
+                    int(val) for val in row[2].strip('[]').split(',')
+                ]
```

Comment:
Reading this list comprehension made me think of another option, `ast.literal_eval`

```python
In[17]: s = "[1, 2, 3, 4]"
In[18]: [int(val) for val in s.strip('[]').split(',')]
Out[18]: [1, 2, 3, 4]
In[19]: from ast import literal_eval
In[20]: literal_eval(s)
Out[20]: [1, 2, 3, 4]
```
I still think this list comprehension better adheres to `"Explicit is better than implicit"` though!